cmd/neofs-adm/internal/modules/morph/config.go

```diff
@@ -223,3 +225,85 @@ func dumpNetworkConfig(cmd *cobra.Command, _ []string) error {
 func invalidConfigValueErr(key []byte) error {
 	return fmt.Errorf("invalid %s config value from netmap contract", key)
 }
+
+func setConfigCmd(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return errors.New("empty config pairs")
+	}
+
+	wCtx, err := newInitializeContext(cmd, viper.GetViper())
+	if err != nil {
+		return fmt.Errorf("can't to initialize context: %w", err)
+	}
+
+	cs, err := wCtx.Client.GetContractStateByID(1)
+	if err != nil {
+		return fmt.Errorf("can't get NNS contract info: %w", err)
+	}
+
+	nmHash, err := nnsResolveHash(wCtx.Client, cs.Hash, netmapContract+".neofs")
+	if err != nil {
+		return fmt.Errorf("can't get netmap contract hash: %w", err)
+	}
+
+	// In NeoFS this is done via Notary contract. Here, however, we can form the
+	// transaction locally.
+	bw := io.NewBufBinWriter()
+	for _, arg := range args {
+		k, v, err := parseConfigValue(arg)
+		if err != nil {
+			return err
+		}
+
+		emit.AppCall(bw.BinWriter, nmHash, "setConfig", callflag.All, nil, k, v)
```

Comment:
Could you add the comment about the `nil`? Probably the comment above can be moved here.